apps/wallet/src/ui/app/pages/home/transfer-coin/index.tsx

```diff
@@ -71,26 +62,18 @@ function TransferCoinPage() {
     const [currentStep, setCurrentStep] = useState<number>(DEFAULT_FORM_STEP);
     const [formData] = useState<FormValues>(initialValues);
 
-    const intl = useIntl();
+    const [coinDecimals] = useCoinDecimals(coinType);
+
     const validationSchemaStepOne = useMemo(
         () =>
             createValidationSchemaStepOne(
                 coinType || '',
                 coinBalance,
                 coinSymbol,
                 gasAggregateBalance,
-                totalGasCoins,
-                intl,
-                balanceFormatOptions
+                coinDecimals
             ),
-        [
-            coinType,
-            coinBalance,
-            coinSymbol,
-            gasAggregateBalance,
-            totalGasCoins,
-            intl,
-        ]
+        [coinType, coinBalance, coinSymbol, coinDecimals, gasAggregateBalance]
```

Comment:
Can't comment on [this line](https://github.com/MystenLabs/sui/pull/5401/files#diff-6c318541d2fa1f39436eedf41e414ea074ecfe216bf0f50501797f2037152fd2R100) but we need to convert it to mist and then to big int

Reply:
Updated to convert to mist and then bigint.